Panic Error when binary ran by a user has no context.

When the user calls the binary directly, a panic error is shown. 

```
panic: runtime error: index out of range [-1]

goroutine 1 [running]:
main.main()
	/home/runner/work/git-ssh-sign/git-ssh-sign/cmd/ssh-sign/main.go:37 +0x2ac
```

This should be more elegant and feedback to the user this is a binary intended to be ran by `git` itself, not the user.

--- a/cmd/ssh-sign/main.go
+++ b/cmd/ssh-sign/main.go
@@ -33,6 +33,10 @@ func main() {
 	*/
 
 	args := os.Args
+	if len(args) == 1 {
+		fmt.Println("This program is not intended to be run directly. It is called by git when signing commits.")
+		os.Exit(1)
+	}
 	commitToSign := args[len(args)-1]
 	sshUID := args[len(args)-2]
 

